indexer/src/main/java/au/org/aodn/esindexer/utils/MapperUtils.java

```diff
@@ -410,4 +422,90 @@ public static Optional<Contacts> mapContactInfo(List<CIContactPropertyType2> con
         }
     }
 
+    public static List<ContactsModel> mapContactsFromOrg(CIResponsibilityType2 ciResponsibility, CIOrganisationType2 organisation) {
+
+        Optional<MapperUtils.Contacts> org = MapperUtils.mapContactInfo(organisation.getContactInfo());
+        return new ArrayList<>(organisation
+                .getIndividual()
+                .stream()
+                .map(individual -> {
+                    ContactsModel invContactsModel = ContactsModel.builder().build();
+                    invContactsModel.setName(MapperUtils.mapContactsName(individual));
+                    invContactsModel.setPosition(MapperUtils.mapContactsPosition(individual));
+                    invContactsModel.setRoles(MapperUtils.mapContactsRole(ciResponsibility));
+                    invContactsModel.setOrganization(organisation.getName().getCharacterString().getValue().toString());
```

Comment:
will there be any cases that organisation object has missing name field or any related parts of the name field?